packages/react-components/src/menu/src/MenuTrigger.tsx

```diff
@@ -71,10 +78,10 @@ export function InnerMenuTrigger({
     children,
     forwardedRef,
     ...rest
-}) {
+}: InnerMenuTriggerProps) {
     const [focusTargetRef, setFocusTarget] = useRefState(null);
 
-    const handleOpenChange = useChainedEventCallback(onOpenChange, (event, isVisible) => {
+    const handleOpenChange = useChainedEventCallback(onOpenChange, (_event, isVisible) => {
```

Comment:
Why the "_" ?

Reply:
In our TSConfig, we added the noUnusedParameter: true. 

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals

Parameters declaration with names starting with _ are exempt from the unused parameter checking. e.g.:


This rules helps maintain a cleaner codebase, notifying you when you can remove a parameter from a function. In this case, we have to keep it since we need it to match a onChange event handler. but on custom function, it's easier to notify and remove them